### PR TITLE
Bump rbpf version to 0.8.5

### PR DESCRIPTION
**Pull request by willhickey**
#### Summary of Changes

Fixes #

### CODE REVIEW

**Review by Lichtso — APPROVED** [review]

**Review by t-nelson — APPROVED** [review]

File: Cargo.toml
Comment (PhilEngel) [inline Cargo.toml:1]: stuggling with an error, any advice would be greatly appreciated: error: failed to select a version for the requirement `solana_rbpf = "=0.8.0"`
candidate versions found which didn't match: 0.8.5, 0.8.4, 0.8.3, ...
location searched: crates.io index
required by package `solana-program-test v1.17.23`
The cargo.toml - 
[dependencies]
anchor-lang = "0.28.0"
solana-program = "1.16.0"
anchor-spl = "0.28.0"
scroll = "0.12.0"
solana_rbpf = "=0.8.5"

[dev-dependencies]
solana-program-test = "=1.17.23"
solana-sdk = "1.16.0"

Comment (Lichtso) [reply Cargo.toml:1]: We yanked v0.8.0 from crates.io as it contained a security flaw.

Comment (willhickey) [reply Cargo.toml:1]: v1.17 isn't supported anymore... upgrading to the latest v1.18 version (currently v1.18.23) will resolve this.

If for some reason you need to use v1.17 then you'll need to update the solana_rbpf dependency. `v0.8.3` is equivalent to `v0.8.0`, but with the security issue patched.

If this doesn't make sense let us know what you're trying to build and we can offer more specific guidance.

Comment (PhilEngel) [reply Cargo.toml:1]: all updates have been done but the error when running cargo update to "solana_rbpf = "=0.8.0" stays, other than changing the cargo.toml, what else needs to be done? (sorry a bit of a newbie..)

**Pull request merged**